### DISCUSSION TITLE
Prusa Slicer errors

### DISCUSSION THREAD

**Discussion by s3ntech** [bassamanator/Sovol-SV06-firmware#24] [General]
Hello,
I am running Klipper on an SV06 and it has been running great using super slicer. Although now that I am looking to do a vase mode print I have been having some troubles "it crashes once vase mode option selected"  and it appears to be a common problem for people. so, I had ventured to try and use Prusa Slicer in order to get my print complete. I had followed "Ehmc130's" [guide ](https://www.reddit.com/r/Sovol/comments/11ov0pv/sovol_sv06_prusa_slicer_profile_recommendations/) and config file to setup the slicer, although i am using the alpha 6 with the Klipper flavor, so i had selected it. 

anyway, once i start a print in Klipper & it heats up, as soon as it goes to start i get the following error "Move out of range: 84.500 -3.000 10.000 [0.000]" 

I am not sure if it is the custom g-code in the loaded profile, or if there is something else i am missing. 3d printing is new to me as of two weeks ago and I'm not sure where to begin. any help is appreciated, Thank you.

**Comment by bassamanator** [comment]
You know I wrote a long post and then decided to scrap it. Let me try this again.

First things first, if using Klipper, you will have to adjust your start and end gcode in your preferred slicer, please see relevant section in the `README.md`. Perhaps this will help your 'out of range' issue?

It appears that you're using pre-release software here, both with PS and SS (SS vase mode only appears to crash in the pre-release version based on my search on their Github), is there any particular reason why? 

Since you are a beginner, let me say that you absolutely should not be using pre-release slicers. The reason being, assigning blame becomes difficult, thus the solution is more difficult to grasp. Pre-release 3d printing software also is more costly when failure occurs: you not only waste all that print time, you also waste all that material, and you create more waste. Having said all that, there are absolutely legitimate reasons to use pre-release slicers, but that is a topic for advanced users, or users who are already comfortable in this hobby, not for beginners who are just beginning to dip their toes in this hobby.

Based on my assumption that you are not using the [latest official release](https://github.com/supermerill/SuperSlicer/releases/tag/2.4.58.5), I would recommend that you download said release, import the SS config bundle found on this repo, and try to print again. I can also tell you that I'm not a beginner, that I use only SS's official release, that I am obsessed with print [quality](https://github.com/bassamanator/everything-sovol-sv06/blob/main/images/prints/octahedroflake1.jpg), and that my setup is so reliable that I can start a print remotely and be 99.99% sure that there will be no issues.

**Reply by s3ntech** [reply]
Thank you for your reply.

I definitely have a lot to learn in this hobby, once I had gone from SS 2.5 to 2.4 and imported the previous profile I created, the error I was having is no longer an issue. thank you for your recommendation.  By the way your print looks amazing, very clean!

**Reply by bassamanator** [reply]
Great! If your issue has been resolved, please close this discussion. It will still be available for other users to read.
